.github/workflows/nglint.yml

```diff
@@ -0,0 +1,45 @@
+name: NgLint
+
+on: 
+  pull_request:
+    branches:
+      - master
+
+
+jobs:
+  nglint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+    - name: Git checkout
+      uses: actions/checkout@v2
+          
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Npm install and test
+      working-directory: client
+      run: |         
+        npm i
+        npm i -g @angular/cli
```

Comment:
Why don't you install the dependencies before as you did in the pylint workflow to be consistent.

Reply:
I install npm packages here, because it's actually needed to run the angular linter.